code/AssetLib/FBX/FBXImporter.cpp

```diff
@@ -90,10 +90,7 @@ static const aiImporterDesc desc = {
 
 // ------------------------------------------------------------------------------------------------
 // Constructor to be privately used by #Importer
-FBXImporter::FBXImporter() :
-        mSettings() {
```

Comment:
`FBX::ImportSettings` has a user-defined default c’tor and is therefore always default-initialized